pyproject.toml

```diff
@@ -4,23 +4,24 @@ description = "Collaborative Brain Wave Analysis Pipeline (Cobrawap)"
 version = "0.2.0"
 readme = "README.rst"
 authors = [
-    { name = "Cobrawap authors and contributors", email = "robin.gutzen@outlook.com" }
+    { name = "Cobrawap authors and contributors" }
 ]
 
 requires-python = ">=3.8"
 dependencies = [
-    "ruamel.yaml >= 0.18.3",
+    "ruamel.yaml == 0.17.32",
     "jinja2 >= 2.10.3",
     "scipy >= 1.7.3",
     "pygments >= 2.4.2",
     "snakemake >= 7.10.0, < 8.0.0",
     "h5py",
     "shapely",
     "elephant >= 1.0.0",
-    "neo >= 0.10.2, < 0.12",
```

Comment:
loading a test .nix file with the load_neo function seems to work with and without a '|' in the file name